grails-app/controllers/streama/VideoController.groovy

```diff
@@ -240,10 +242,20 @@ class VideoController {
 
   def getErrorReports () {
     def reports = Report.findAllByResolved(false)
-    respond reports
-    return
+      return [reportList:reports]
+  }
+
+  @Transactional
+  def resolveReports() {
+    def jsonData = request.JSON
+    def resolvedReports = []
+    jsonData.ids.each { id ->
+      def report = Report.get(id)
+      report.resolved = true
+      report.lastUpdated = new Date()
```

Comment:
Don't I? I thought lastUpdated was automatically set when the report was created, but not when updated right? Otherwise dateCreated would also be updated here.

Reply:
the logic is: dateCreated AND lastUpdated are automatically set when first INSERTing the row, and lastUpdated is set on all subsequent updates.